src/applications/disability-benefits/686/helpers.jsx

```diff
@@ -104,14 +98,13 @@ export const otherRelationshipDescription = (
   <div className="usa-alert usa-alert-info no-background-image">You’re applying on <strong>behalf</strong> of the Veteran who’s sponsoring this application. First, we’ll ask for your information as the applicant. Then, we’ll ask for the Veteran's information as the sponsor.</div>
 );
 
-export function isNotVeteran(item) {
-  return get('view:relationshipToVet', item) !== '1';
+export function isVeteran(item) {
+  const relationship = get('view:relationshipToVet', item);
+  return relationship && relationship === '1';
 }
 
-export function isSpouse(item) {
-  return get('application.claimant.relationshipToVet', item) === '2';
+export function isNotVeteran(item) {
```

Comment:
You could have this function call `isVeteran` and negate the output, to reduce the duplication.